lib/loc/make.go

```diff
@@ -30,13 +30,23 @@ import (
 //  - if it's in the 'name:ver' format, returns locator with system repo (systemrepo/name:ver)
 //  - if it's in the 'name' format, returns locator with system repo and latest meta-version (systemrepo/name:0.0.0+latest)
 func MakeLocator(app string) (*Locator, error) {
+	return MakeLocatorWithDefault(app, "")
+}
+
+// MakeLocatorWithDefault is like MakeLocator but uses the provided default
```

Comment:
nit: I think MakeXXX is non-consistent with the rest of the constructors - would be more consistent to use `NewLocatorWihDefault` (i.e. prefix with `New` for new APIs)